tox.ini

```
@@ -14,4 +14,6 @@ basepython =
 
 deps =
     django4.2: Django>=4.2,<5.0
+    django5.0: Django>=5.0,<5.1
     wagtail5.2: wagtail>=5.2,<6.0
+    wagtail6.0: wagtail==6.0rc1
```

I tend to define this as
```suggestion
    wagtail6.0: wagtail>=6.0rc1,<6.1
```

Which means it will work after the full release is out, or if we have an rc2 without need to change loads